quint/package.json

```diff
@@ -84,10 +87,14 @@
   },
   "devDependencies": {
     "@types/chai": "^4.2.18",
+    "@types/gunzip-maybe": "^1.4.0",
     "@types/json-bigint": "^1.0.1",
     "@types/lodash.isequal": "^4.5.6",
     "@types/mocha": "^8.2.3",
     "@types/node": "^18.16.0",
+    "@types/node-fetch": "^2.6.4",
+    "@types/tar": "^6.1.5",
+    "@types/tar-fs": "^2.0.1",
```

Comment:
Again, assuming these are not needed:

```suggestion
```
Also the gunzip above.

Reply:
d2045ae9